persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/ClientStateWithPublicAddress.java

```diff
@@ -12,11 +12,13 @@
 public class ClientStateWithPublicAddress extends ClientState {
 
   private final InetSocketAddress publicAddress;
+  private final boolean internal;
```

Comment:
Probably implied by type `boolean`, but maybe should be named `isInternal`?
(and/or add brief javadoc comment on its purpose)